lib/routes/account.js

```diff
@@ -1113,13 +1115,18 @@ module.exports = function (
               push.notifyUpdate(uid, 'accountVerify')
 
               return db.verifyEmail(account)
-                .then(mailer.sendPostVerifyEmail.bind(
-                    mailer,
-                    account.email,
-                    {
-                      acceptLanguage: request.app.acceptLanguage
+                .then(
+                  function() {
+                    // Our post-verification email is very specific to sync.
+                    if (! service || service === 'sync') {
```

Comment:
Indeed - no service currently sends this parameter, so if it's missing then we have to assume that this could be sync.  I will add a comment to clarify